packages/sanes-chrome-extension/src/routes/login/index.tsx

```diff
@@ -0,0 +1,6 @@
+import * as React from 'react';
+import Layout from './components';
+
+const Login = (): JSX.Element => <Layout />;
```

Comment:
Remember now using only functional components, we need to balance the components I expect to have here, the validation function, onLogin function and the main layout.

```
function onValidate() {
    ...
}

function onLogin() {
   ...
}

const Login = () => {
    .... 
   return (
    <PageLayout id={LOGIN_ROUTE} primaryTitle="Log" title="In">
      <Form onSubmit={handleSubmit}>
         <LoginFormFields ... />
         </LoginFormControls ... />
     </Form>
   </PageLayout>

export default Login
```